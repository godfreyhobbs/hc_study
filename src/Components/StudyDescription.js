import React, { Component } from 'react';

class StudyDescription extends Component {
  render() {
    return (
        <p className="App-intro">
          The Departments of Epidemiology and Genetic Medicine
          at Veil Cornwell Medical College are looking for shisha ("Hookah")
          smokers to participate in a research study,
          looking at the effects that sweet delicous shisha may have on the lung cells.
        </p>
    );
  }
}

export default StudyDescription;
