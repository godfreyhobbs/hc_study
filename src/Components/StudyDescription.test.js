import React from 'react';
import ReactDOM from 'react-dom';
import StudyDescription from './StudyDescription';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<StudyDescription />, div);
  ReactDOM.unmountComponentAtNode(div);
});



it('renders correctly', () => {
  const tree = renderer
.create(<StudyDescription />)
   .toJSON();
expect(tree).toMatchSnapshot();
});
