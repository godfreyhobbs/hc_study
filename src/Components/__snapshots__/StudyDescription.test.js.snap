// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="App-intro"
>
  The Departments of Epidemiology and Genetic Medicine at Veil Cornwell Medical College are looking for shisha ("Hookah") smokers to participate in a research study, looking at the effects that sweet delicous shisha may have on the lung cells.
</p>
`;
